scripts/build-packages.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+const fs = require("fs");
+const shell = require("shelljs");
+
+shell.cd(path.join(__dirname, ".."));
```

Comment:
What's this for?

I gave it a try, and it can be deleted.

Reply:
To set the cwd so that the shelljs has the right context. This is mainly for preventing shell command bugs.

Reply:
Using `yarn workspace` should avoid this problem

Reply:
Add a note here to explain

Reply:
There is no problem in the PR either. It's not about a specific command but to prevent other shelljs issues should more commands be added to this in the future. Think of it as a safe guard ritual. Yes it can be deleted now but leave it there would do no harm.

Reply:
OK, Then add a note here to explain the situation

Reply:
Done.